core/commands/bitswap.go

```diff
@@ -6,17 +6,18 @@ import (
 	"io"
 
 	cmds "github.com/ipfs/go-ipfs/commands"
+	e "github.com/ipfs/go-ipfs/core/commands/e"
```

Comment:
I would rename it to "err" or "error" to make it more obvious what it is.  @whyrusleeping agree?